### PR TITLE
Added Navbar

### PR DESCRIPTION
**Pull request by Raghav888**
Added Navbar
Added Responsiveness

### CODE REVIEW

File: documentation/document.css [hunk 1]
```diff
@@ -197,6 +198,14 @@ html {
     border: 1px solid var(--primary-color);
 }
 
+.component-preview-container-nav {
+    max-width: 100%;
+    margin: 2rem 0;
+    /* padding: 0.5rem; */
```
Comment (tusharkandpal) [inline documentation/document.css:204]: remove this comment

**Review by tusharkandpal — APPROVED** [review]
one small change ...else looks good to merge! 👍🏻

File: documentation/document.css (hunk 1 above)
Comment (Raghav888) [reply documentation/document.css:204]: Sure will do it

**Review by harshitadaswani — APPROVED** [review]
You can go ahead and merge it

**Pull request merged**